interface-definitions/load-balancing-wan.xml.in

```diff
@@ -0,0 +1,395 @@
+                  <leafNode name="resp-time">
+                    <properties>
+                      <help>Ping response time (seconds)</help>
+                      <valueHelp>
+                        <format>u32:1-30</format>
+                        <description>Response time (seconds)</description>
+                      </valueHelp>
+                      <constraint>
+                        <validator name="numeric" argument="--range 1-30"/>
+                      </constraint>
+                    </properties>
+                  </leafNode>
+                  <leafNode name="target">
+                    <properties>
+                      <help>Health target address</help>
+                      <valueHelp>
+                        <format>ipv4</format>
+                        <description>Health target address</description>
+                      </valueHelp>
+                      <constraint>
+                        <validator name="ipv4-address"/>
+                      </constraint>
+                    </properties>
+                  </leafNode>
+                  <leafNode name="test-script">
+                    <properties>
+                      <help>Path to user defined script</help>
+                      <valueHelp>
+                        <format>txt</format>
+                        <description>Script in /config/scripts</description>
+                      </valueHelp>
+                      <constraint>
+                        <validator name="script"/>
+                      </constraint>
+                    </properties>
+                  </leafNode>
+                  <leafNode name="ttl-limit">
+                    <properties>
+                      <help>Ttl limit (hop count)</help>
+                      <valueHelp>
+                        <format>u32:1-254</format>
+                        <description>Number of hops</description>
+                      </valueHelp>
+                      <constraint>
+                        <validator name="numeric" argument="--range 1-254"/>
+                      </constraint>
+                    </properties>
+                  </leafNode>
+                  <leafNode name="type">
+                    <properties>
+                      <help>WLB test type</help>
+                      <completionHelp>
+                        <list>ping ttl user-defined</list>
+                      </completionHelp>
+                      <valueHelp>
+                        <format>ping</format>
+                        <description>Test with icmp echo response</description>
+                      </valueHelp>
+                      <valueHelp>
+                        <format>ttl</format>
+                        <description>Test with udp TTL expired response</description>
+                      </valueHelp>
+                      <valueHelp>
+                        <format>user-defined</format>
+                        <description>User defined test script</description>
```

Comment:
User-defined (compound adjective should use a hyphen).

Reply:
Done!